configure.ac

```diff
@@ -770,7 +818,8 @@ AC_MSG_RESULT([  Log directory            = ${LOGDIR}])
 AC_MSG_RESULT([  Log rotate directory     = ${LOGROTATEDIR}])
 AC_MSG_RESULT([  corosync config dir      = ${COROSYSCONFDIR}])
 AC_MSG_RESULT([  init config directory    = ${INITCONFIGDIR}])
-AC_MSG_RESULT([  Features                 =${PACKAGE_FEATURES}])
+AC_MSG_RESULT([  Features                 = ${PACKAGE_FEATURES}])
```

Comment:
I know this looks weird in source code but previous version is actually correct. PACKAGE_FEATURES is concatenated in the following way `PACKAGE_FEATURES="$PACKAGE_FEATURES debug"` so it will always contain beginning space - the reason why the space is not added in the features line